server/project/api/models.py

```diff
@@ -24,10 +55,31 @@ class Timezone(db.Model):
     minutes = db.Column(db.Integer)
     dst_hours = db.Column(db.Integer, nullable=False)
     dst_minutes = db.Column(db.Integer)
-    
+
     def __init__(self, name, hours, minutes, dst_hours, dst_minutes):
         self.name = name
         self.hours = hours
         self.minutes = minutes
         self.dst_hours = dst_hours
         self.dst_minutes = dst_minutes
+
+
+class Event(db.Model):
+    __tablename__ = 'events'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(32), nullable=False)
+    location = db.Column(db.String(256))
+    description = db.Column(db.String(1024))
+    url = db.Column(db.String(32), nullable=False, unique=True)
+    colour = db.Column(db.String(6), nullable=False)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
+    availability = db.relationship('Availability', backref='events')
+
+    def __init__(self, name, location, description, url, colour, user_id):
+        self.name = name
+        self.location = location
+        self.description = description
+        self.url = url
+        self.colour = colour
```

Comment:
just a little comment, to stay consistent with canadian and american selling :P as long as the team is aware i'm sure it's fine

Reply:
Thank you I will swap to American spelling since it is easier for me to change my work than to make the other 2 change theirs.